src/svcutilcore/src/Microsoft/Tools/ServiceModel/SvcUtil/ToolRuntime.cs

```diff
@@ -31,29 +31,13 @@ internal ToolExitCodes Run()
                 InputModule inputModule = InputModule.LoadInputs(_options);
 
                 Tool.Assert(_options.GetToolMode().HasValue, System.SR.Format(System.SR.AmbiguousToolUseage, Options.Cmd.Target, Options.Cmd.Validate));
-                ToolMode toolMode = _options.GetToolMode().Value;
-
-                return ExecuteToolMode(toolMode, inputModule);
+                return ExecuteToolMode(inputModule);
```

Comment:
Do we want to keep the ExecuteToolMode still taking two params for potentially adding more mode in future?

Reply:
We can easily add it back if it is necessary in the future. But we don't have any plan or thoughts now.